pyproject.toml

```diff
@@ -108,6 +108,7 @@ docs = [
     "myst-parser == 0.18.0",
     "nlp_primitives >= 2.3.0",
     "autonormalize >= 2.0.1",
+    "click >=7.0.0",
```

Comment:
need for https://github.com/alteryx/featuretools/blob/main/docs/notebook_version_standardizer.py